Lib/test/test_inspect.py

```diff
@@ -562,6 +576,14 @@ def test_range_toplevel_frame(self):
     def test_range_traceback_toplevel_frame(self):
         self.assertSourceEqual(mod.tb, 1, None)
 
+    def test_class_frame(self):
+        self.assertSourceEqual(mod.A.fr, 100, 103)
+        self.assertNotSourceEqual(mod.A.fr, 85, 86)
```

Comment:
What is the motivation for the assertNotSourceEqual tests? If it's equal to a range, surely you don't have to also test that it's unequal to another range?